src/app/cluster/cluster-details/edit-cluster/component.ts

```diff
@@ -120,6 +124,14 @@ export class EditClusterComponent implements OnInit, OnDestroy {
       }
       this.form.updateValueAndValidity();
 
+      if (this._settings.konnectivity.enabled) {
+        this.form.get(Controls.Konnectivity).setValue(true);
+      }
+      if (this._settings.konnectivity.enforced) {
```

Comment:
From your pinned PRs I can't see, that you've added this options (enable+enforce) to the api settings spec.
If they're missing, this will not do anything.
If it is added in backend, than you also might want to add to our Admin Panel, so that admins can change it from ui and specify the default behaviour of the konnectivity checkbox.

Reply:
Yeah, looks like some logic might be missing here.

Reply:
Yep. They are not added in the backend. 

I wasn't sure if we wanted them at all.

I will remove them for now and do admin enforcement stuff can be done in another PR?

Reply:
sure